tensorflow/tsl/platform/windows/windows_file_system.cc

```diff
@@ -546,13 +546,36 @@ Status WindowsFileSystem::DeleteDir(const string& name,
   return result;
 }
 
+static std::wstring GetSymbolicLinkTarget(const std::wstring& linkPath) {
+  TCHAR path[MAX_PATH];
+  HANDLE hFile = ::CreateFileW( linkPath.c_str(),
+    0,
+    0,  
+    0,
+    OPEN_EXISTING,
+    FILE_FLAG_BACKUP_SEMANTICS,
+    0);
+  if (INVALID_HANDLE_VALUE != hFile) {
```

Comment:
Nit: in general it reduces indentation and simplify the control flow to do an early-exit, if you reverse the condition you get:
```
if (INVALID_HANDLE_VALUE == hFile) 
  return {};

auto rcode = GetFinalPathNameByHandle(hFile, path, MAX_PATH, FILE_NAME_NORMALIZED);
if (!rcode)
   return {}

...

```